client/src/components/parent-signup-form/parent-signup.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+
+class ParentSignup extends React.Component {
+  state = {
+   name: '', password: '', phone: '', email: '', location: ''}
+
+    handleChange = event => {
+     const target = event.target;
+     const value = target.type === 'checkbox' ? target.checked : target.value;
+     console.log("Typed: " + target.name, value);
+     this.setState({[target.name]: value});
+   };
+
+    handleSubmit = event => {
+     event.preventDefault();
+     const data = JSON.stringify(this.state);
+     console.log("(Fake) submitting data: ", data)
+     this.setState({ name: '', password: '', phone: '', email: '', location: ''});
+   };
+
+    render() {
+     return (
+       <form onSubmit={this.handleSubmit}>
```

Comment:
I would look into a react form framework for this to clean up the handle change events, [formik](https://github.com/jaredpalmer/formik) be very helpful!